### PR TITLE
Feat/126 update file api

### PR DESCRIPTION
**Pull request by saneDG**
Add file-update endpoint and lambda function.

Updates file content while keeping same id. File name can be edited in same request with `name` parameter.

`PUT` `/api/alfresco/file/{folder}/{nodeId}`

### CODE REVIEW

File: packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts [hunk 1]
```diff
@@ -36,6 +41,15 @@ export class AlfrescoFileRequestBuilder {
     } as unknown as RequestInit;
     return options;
   }
+  public async updateRequestBuilder(event: ALBEvent, headers: HeadersInit) {
+    const buffer = base64ToBuffer(event.body as string);
+    const options = {
+      method: 'PUT',
+      body: buffer,
+      headers: headers,
+    } as unknown as RequestInit;
```
Comment (mehtis) [inline packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts:50]: Curious about the unknown here. Got anything to share?

**Review by mehtis — APPROVED** [review]

File: packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts (hunk 1 above)
Comment (saneDG) [reply packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts:50]: I don't know reason for this either 😕 Must be unnecessary copypaste mistake. I'll remove this before merge.

Comment (saneDG) [reply packages/server/lambdas/alfresco/fileRequestBuilder/alfrescoRequestBuilder.ts:50]: This is actually fixed in #163

File: packages/server/lambdas/alfresco/update-file.ts [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    const res = await fetch(url, options);
+    const result = (await res.json()) as AlfrescoResponse;
+    return result;
+  } catch (err) {
+    console.error('error:' + err);
+  }
+};
+
+/**
+ * Upload custom content for page. Example request: /api/alfresco/file/linjakaaviot
```
Comment (NinaDang97) [inline packages/server/lambdas/alfresco/update-file.ts:38]: Should be "Update content"? Could you update this comment section?

**Review by NinaDang97 — APPROVED** [review]

**Pull request merged**